pkg/apis/edge/v1alpha1/edge-placement.go

```diff
@@ -107,12 +104,19 @@ type NonNamespacedObjectReferenceSet struct {
 	// Empty list means nothing matches.
 	Resources []string `json:"resources"`
 
-	// `resourceNames` is a list of objects that match by name.
+	// `namespaceSelectors` tests the labels on the object's namespace.
+	// A non-namespaced object matches if and only if this list is empty.
+	// A namespaced object matches if and only if the namespace's labels match
+	// at least one member of this list.
+	NamespaceSelectors []metav1.LabelSelector `json:"namespaceSelectors"`
+
+	// `objectNames` is a list of object names that match.
 	// An entry of `"*"` means that all match.
 	// Empty list means nothing matches.
-	ResourceNames []string `json:"resourceNames,omitempty"`
+	ObjectNames []string `json:"objectNames,omitempty"`
 
 	// `labelSelectors` allows matching objects by a rule rather than listing individuals.
+	// An object maches if and only if its labels match at least one member of this list.
 	LabelSelectors []metav1.LabelSelector `json:"labelSelectors,omitempty"`
 }
 
```

Comment:
Now that we can select specific API groups and versions, how do we ensure that the resources groups and versions are included in the list of resource groups and versions that the syncer is configured to sync ?

Reply:
The EdgePlacementSpec does not allow the author to be specific about API version.
The placement translator remains responsible for maintaining the translation of the collection of EdgePlacement objects into the collection of SyncerConfig objects.